jetbrains-core/src/software/aws/toolkits/jetbrains/core/gettingstarted/SetupAuthenticationDialog.kt

```diff
@@ -274,6 +275,7 @@ class SetupAuthenticationDialog(
             textField()
                 .comment(message("gettingstarted.setup.idc.profile.comment"))
                 .errorOnApply(message("gettingstarted.setup.error.not_empty")) { it.text.isBlank() }
+                .errorOnApply(message("gettingstarted.setup.idc.profile.error_message")) { ProfileFile.defaultProfileFile().profiles().containsKey(it.text) }
```

Comment:
this isn't quite right, could you add tests?

Reply:
sure, definitely

Reply:
As discussed offline with Richard, suggested that should only be looking at SSO sessions name. It's not about whether there's a profile with the same name; it's about finding an SSO session with a matching name.